packages/rum/src/domain/record/privacy.ts

```diff
@@ -197,7 +195,6 @@ export function getTextContent(
   } else if (shouldMaskNode(textNode, nodePrivacyLevel)) {
     if (isStyle) {
       // Style tags are `overruled` (Use `hide` to enforce privacy)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** avoid the empty `if`:
```ts
  } else if (
    shouldMaskNode(textNode, nodePrivacyLevel) &&
    // Style tags are `overruled` (Use `hide` to enforce privacy)
    !isStyle
  ) {
```

or maybe move this condition to `shouldMaskNode`?